docs/index.rst

```diff
@@ -8,12 +10,34 @@ Vulcanexus Documentation
   :alt: Vulcanexus
   :target: http://www.vulcanexus.com/
 
-.. Vulcanexus introduction
 
-.. _main_features:
+*Vulcanexus* is an enhanced *ROS 2* (Robot Operating System) distribution.
+It allows users to build robotics applications combining the unique *Vulcanexus* elements with the *ROS 2* libraries,
+having *Fast DDS* as its fixed middleware implementation.
+
+These open source elements include numerous features and tools, providing *Vulcanexus* users customizable solutions
+while improving overall system performance. With *Vulcanexus*, users have fast access to constantly improving
+functionalities, such as the latest *Fast DDS* version along with its new features.
+
+*Vulcanexus* combinable elements are:
+
+1. **VULCANEXUS-CORE**: a set of software libraries that enables users to build the most comprehensive and
+   straightforward robotics application.
+   Consists of eProsima *Fast DDS* and *ROS 2*.
+2. **VULCANEXUS-TOOLS**: a set of features and applications which allows users to test, improve and configure the
+   performance of *Vulcanexus* in their systems.
+3. **VULCANEXUS-MICRO**: provides access for resource constrained devices (micro-controllers) to the DDS world, bridging
+   the gap between them and *ROS 2*.
+4. **VULCANEXUS-CLOUD**: scales and integrates *ROS 2* networks located in geographically spaced environments, and
+   enables the deployment of DDS entities in the cloud in a quick and easy way.
+
+*Vulcanexus* created a collection of downloadable packages that include useful combinations of the previously described
+elements with *ROS 2*:
+
+.. figure:: /rst/figures/intro/vulcanexus_packages_table.svg
 
-Main features
--------------
+The following documentation describes each *Vulcanexus* package in more detail, its functionalities and its application
+to the users' system architecture.
```

Comment:
This is not true right now. Either adding a note stating that the documentation is still under work or I suggest removing this sentence.

Another option is modifying to state what the documentation really details:

```suggestion
The following documentation describes how to install *Vulcanexus* packages, some tutorials to know how to get started, the supported platforms and releases.
```